source/yellow/index.html

```diff
@@ -315,6 +310,16 @@ <h1>Buying a Raspberry Pi Compute Module 4 (CM4)</h1>
   </ul>
 </div>
 
+<h1 class="sub-title">FAQs</h1>
+<div class="faq-list">
+  {% details "Can I buy the Home Assistant Yellow pre-assembled?" %} <p>No. We previously offered a Home Assistant Yellow Standard model that came pre-assembled with a Raspberry Pi CM4, but this model is no longer manufactured. It is now recommended that if you want a more plug-and-play experience, you purchase a <a href="https://www.home-assistant.io/green" target="_blank">Home Assistant Green</a>, as it requires no assembly and matches many of the capabilities of the Yellow, especially if you add a <a href="https://www.home-assistant.io/connectzbt1/" target="_blank">Home Assistant Connect ZBT-1</a> to give it Zigbee/Thread capability.
+  </p>
+  {% enddetails %}
+
+  {% details "What happened to Home Assistant Yellow Standard, and will it still be supported?" %} <p>The Home Assistant Yellow Standard was originally designed to be the beginner option in our product line, as it was pre-assembled and easier to get started with. In 2023, we launched the <a href="https://www.home-assistant.io/green" target="_blank">Home Assistant Green</a>, which offered similar features, including easy setup, but at a lower price point. This led us to end the manufacture of the Home Assistant Yellow Standard and focus on Yellow Kit versions for advanced users. Rest assured, as the Home Assistant Yellow Standard hardware is identical to our Kits, it will continue to receive the same support and software updates.
```

Comment:
```suggestion
  {% details "What happened to Home Assistant Yellow Standard, and will it still be supported?" %} <p>The Home Assistant Yellow Standard was originally designed to be the beginner option in our product line, as it was pre-assembled and easier to get started with. In 2023, we launched the <a href="https://www.home-assistant.io/green" target="_blank">Home Assistant Green</a>, which offered similar features, including easy setup, but at a lower price point. This led us to end the manufacture of the Home Assistant Yellow Standard and focus on Yellow Kit versions for advanced users. Rest assured, it will continue to receive the same support and software updates as the Yellow Kit variants.
```